kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/Company.java

```diff
@@ -6,11 +6,16 @@
 import java.util.List;
 import java.util.Objects;
 
-
-@NamedQuery(
-        name = "Company.retrieveCompanyByBeginning",
-        query ="FROM Company WHERE SUBSTR(name, 0, 3) = :SEARCHED"
-)
+@NamedQueries({
+        @NamedQuery(
+                name="Company.retrieveCompanyByBeginning",
```

Comment:
w tresci zadania jest pwoiedziane, ze zarowno pracownikow jak i firmy wyszukujemy po jakims fragmencie nazyw :) nawet podano czesc zapytania. to troche co innego, niz bylo w 13.4